x/auth/middleware/static_feemarket.go

```diff
@@ -0,0 +1,70 @@
+package middleware
+
+import (
+	"math"
+
+	codectypes "github.com/cosmos/cosmos-sdk/codec/types"
+	sdk "github.com/cosmos/cosmos-sdk/types"
+	sdkerrors "github.com/cosmos/cosmos-sdk/types/errors"
+)
+
+// StaticFeeMarket implements a static feemarket.
```

Comment:
Also not sure if "static" is the right word, it seems like it's a fixed price per gas across the whole network. Maybe `MinGasPriceFeeMarket`?

Reply:
There is no Fee Market, so I would remove that suffix. Other ideas:
* ValidatorFee
* ValidatorFixedFee

Reply:
if we agree with a change, then we should also rename the file.

Reply:
`ValidatorTxFee`.

Reply:
If we're going with `ValidatorTxFee`, do we still want to keep the whole "FeeMarket" nomenclature used elsewhere in this PR? Specifically the middleware interface.

Reply:
> If we're going with `ValidatorTxFee`, do we still want to keep the whole "FeeMarket" nomenclature used elsewhere in this PR? Specifically the middleware interface.

the interface is still called `FeeMarket`, `ValidatorTxFee` is the default implementation. Since the `ValidatorTxFee` also contains the priority logic, so it's kind of a market too?